src/conf_functions/gene_pickers.rs

```diff
@@ -2,20 +2,20 @@ use crate::R;
 use rand::seq::IteratorRandom;
 use rand::Rng;
 
-pub fn stochastic_pick<'a, const N: usize>(
-    genes: &'a mut [f64; N],
-    domain: &[(f64, f64); N],
+pub fn stochastic_pick<'a, const N: usize, T: Default + Copy + Clone>(
```

Comment:
In `lib.rs` we use `Gene` instead of `Default + Copy + Clone` now, I think it should be used here as well.